script.py

```diff
@@ -89,28 +85,24 @@ def print_vehicle_info(path):
 
 
 def is_valid_bus(user_input, response_data_json):
-    if(
-        user_input.lower() == 'all'
-        or user_input.lower() == 'exit'
-        or user_input.lower() == 'raw'
-    ):
-        return True
-
+    valid_inputs = ['all', 'exit', 'raw']
     for bus in response_data_json:
-        if bus['id'] == user_input:
-            return True
+        valid_inputs.append(bus['id'])
 
-    return False
+    return user_input.lower() in valid_inputs
```

Comment:
Why do you need to call `lower()` again here? I think you've transformed it at line 53.